### DISCUSSION TITLE
AUTH:Received control message:AUTH_FAILED

### DISCUSSION THREAD

**Discussion by SunStarJ** [schwabe/ics-openvpn#1535] [Configuration problems] [answered]
To make issues more manageable, I would appreciate it if you fill out the following details as applicable:

# General information
1. Android 12
2. Android vivo
3. Device    iqoo 9 pro
4. Version of the 0.7.41
 
opvn config :
```
client
dev tun
proto udp
remote 154.8.150.244 1194
resolv-retry infinite
nobind
persist-key
persist-tun
remote-cert-tls server
cipher none
auth none
# comp-lzo only used when enable compress 


# explicit-exit-notify only used when proto is UDP

explicit-exit-notify 1

verb 3
ignore-unknown-option data-ciphers compat-mode
data-ciphers none
compat-mode 2.4.8
```

when i use OpenVpn Connect it can work ,but in Ics OpenVpn "AUTH:Received control message:AUTH_FAILED"

**Comment by schwabe** [comment, marked as answer]
AUTH_FAILED without any reason is hard to diagnose. Check the server log why it refuses the client.
